com.aobuchow.themes.spectrum.preferences/src/com/aobuchow/themes/spectrum/preferences/ColorHSL.java

```diff
@@ -102,6 +102,21 @@ public float getPerceivedLuminance() {
 		return (299f * color.getRed() + 587f * color.getGreen() + 114f * color.getBlue()) / 1000;
 	}
 
+	//TODO: Ensure it's within the range [0, 360]
```

Comment:
For all of these, if the value given is outside of the bounds, set it to the nearest bound. In the future, an exception could be thrown instead.